### PR TITLE
Update rgccacmd to 3.0.1

### PR DESCRIPTION
**Pull request by ecamenen**
Update to fix https://github.com/galaxyproject/tools-iuc/pull/3764

### CODE REVIEW

File: recipes/rgccacmd/meta.yaml [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
   noarch: generic
-  number: 1
+  number: 0
   rpaths:
     - lib/
     - lib/R/lib/
 
 requirements:
   host:
+    - wget
```
Comment (bgruening) [inline recipes/rgccacmd/meta.yaml:21]: you need that in the run section, do you?

Comment (ecamenen) [reply recipes/rgccacmd/meta.yaml:21]: It seems to be the case with my new version. Without it, the last commit failed.

Comment (ecamenen) [reply recipes/rgccacmd/meta.yaml:21]: This was to download the command line interface that was migrated to another directory in the latest version. Once installed during the build, it should be fine, but I can add it in the run section to be sure. Strangely the similar version of conda in my personal directory doesn't need "wget" (https://anaconda.org/icm-iconics/r-rgcca).

**Review by bgruening — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@BiocondaBot please merge

**Pull request merged**